### PR TITLE
IA Updates

### PR DESCRIPTION
**Pull request by alisonjoseph**
#1516 

Reviewers, here is the list of pages with changes:

- Home page: Migration guide tile changed to Carbon events tile
- All about Carbon: existing content from **About Carbon** and **Governance** moved into two new pages. Note: These pages are up for review in the next pass: 
  - What is Carbon?
  - How we work
- Designing
  - Get started (new page, new content)
  - Design kits (originally **Get started** for Design)
  - Other resources (part of original **Resources** section)
- Developing
  - Get started (new page, new content)
  - Frameworks (originally **Get started** for Devs)
  - Developer resources (part of original **Resources** section)
  - Angular, React, Vue tutorials -- new position, no changes to pages or hierarchy
- Contributing (heading changed from **How to Contribute**), no other changes
- What's happening (new section) 
  - Changelog and roadmap (moved from **Updates**)
  - News and articles (moved from **Updates**)
  - Events (new page)
  - Work in progress (no changes to page, moved from **Updates**)
- Help (now includes content from original **Help** and **Support**)
  - FAQs (no changes yet)
  - Certificate of Originality (moved from **About Carbon**)
  - Migration guide (moved from **Updates**)
  - Contact us (adapted from original **Support** page)

### CODE REVIEW

**Comment by jeanservaas** [comment]
Hey guys,

I just had a couple of visual comments:


**Rules**
The rules on both of these pages should be using the (IDL Philosopy > Principles page and the new Events page) $ui-03 token for subtle borders which will give the page an airier, less crammed look. That's a Gray 20, instead of what looks now to be a Gray 50.

image

**Margin bottoms**

The margin bottoms on the Philosoply > Principles page are 56px (combination of margin bottom and padding) — we want to match this, Events page margin bottom padding is tighter, causing a crammed look.

image

image

**H2 / H3 stack**

I don't like the stack of the H2 right on top of the H3 for several reason. 

- We've overriding the spacing convention on the site to do this. Normally, we'd have 48px of space over that H3

- The Accessibility page already has an Events page that we designed for them and it just uses the H2, same with the Philosophy > Principles page.

I would keep our design as Jan had it before, making it an H2 stack. Whether it's two or three lines, or how Jan wants to write it is up to her.

image

image

**Comment by janchild** [comment]
@alisonjoseph for the tiles on the Events page, could you please label them `Email us` and launch an email to carbon@us.ibm.com on click? Thank you!

**Comment by jeanservaas** [comment]
Thanks for turning the changes around so fast!

One other thing that I didn't notice before (don't hate me... it's a hanging grid comment)

When the mini resource cards are not in the margin, but in-line with the body of the page, they are supposed to hang like the large resource cards. Right now they are flush to the columns.

**as is:**

image

**spec** (this is a tablet view cause I had it on hand, but you get the picture... it hangs on both sides.

image

**Comment by alisonjoseph** [comment]
updated ✔️ @jeanservaas @janchild

File: conf.d/rewrite.conf [hunk 1]
```diff
@@ -30,4 +12,31 @@ rewrite /experimental/order-summary-template/(.*) https://pages.github.ibm.com/C
...
+rewrite /updates/whats-new /whats-happening/changelog-and-roadmap permanent;
+rewrite /updates/work-in-progress /whats-happening/work-in-progress permanent;
+
+# Help
+rewrite /updates/migration-guide/(.*) /help/migration-guide/\$1 permanent;
+rewrite /help/support /help/contact-us permanent;
+rewrite /help/faq /help/faqs permanent;
+
+# Developing
+rewrite /get-started/develop/(.*) /developing/frameworks/\$1 permanent;
```
Comment (vpicone) [inline conf.d/rewrite.conf:38]:
```suggestion
rewrite /get-started/develop(.*) /developing/frameworks/\$1 permanent;
```

We  should prob be consistent with how we handle slashes with these.

Comment (alisonjoseph) [reply conf.d/rewrite.conf:38]: don't we need that slash there, so it matches any of the pages that hang off of /develop and redirects them to pages off of /frameworks?

**Comment by jeanservaas** [comment]
image


Just change those dashes to en-dashes between the times (no spaces on either side) por favor.

File: conf.d/rewrite.conf [hunk 2]
```diff
@@ -30,4 +12,31 @@ rewrite /experimental/order-summary-template/(.*) https://pages.github.ibm.com/C
...
+rewrite /resources /developing/developer-resources permanent;
+rewrite /tutorial/angular(.*) /developing/angular-tutorial\$1 permanent;
```
Comment (vpicone) [inline conf.d/rewrite.conf:40]: Sorry, my suggestion as incorrect. What I meant was that we should pick including the slash or not and stick to it. I like leaving it behind like you did for `get-started/develop`.

```suggestion
rewrite /tutorial/angular/(.*) /developing/angular-tutorial/\$1 permanent;
```

Comment (alisonjoseph) [reply conf.d/rewrite.conf:40]: ahh gotcha. I see it now

**Comment by jeanservaas** [comment]
@mjabbink @alisonjoseph 

Mike and I were just talking and he made a good point about the mini cards... they are actually Actions and therefore, the correct component to use would be a button.

This exact point came up with the accessibility team. The mini / resource cards are meant to be on-site or off-site links for browsing, but they don't ask you to perform an action, like "Make a Github issue" or "Send an email"

**Comment by alisonjoseph** [comment]
@jeanservaas which button type would be appropriate here to switch them to?

**Comment by jeanservaas** [comment]
Sorry, @alisonjoseph 

Primary button (default 48px) with icon

**Comment by mjabbink** [comment]
@jeanservaas @alisonjoseph I think the rule needs to be a bit darker.

i.e.
https://www.ibm.com/brand/systems/cloud/brand/strategy

**Comment by jeanservaas** [comment]
This looks great, I'm going to approve!

One more small thing. @mjabbink wants the rule back to the $ui-04 (g50) since it's a section divider it should probably be more prominent. M'bad. Wah wah.

File: src/pages/all-about-carbon/what-is-carbon.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+code, design tools and resources, human interface guidelines, and a vibrant
+community of contributors.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Introduction</AnchorLink>
+<AnchorLink>Guiding principles</AnchorLink>
+<AnchorLink>How we work</AnchorLink>
```
Comment (tw15egan) [inline src/pages/all-about-carbon/what-is-carbon.mdx:22]: There doesn't seem to be a section for this on this page, so this link doesn't work

Comment (janchild) [reply src/pages/all-about-carbon/what-is-carbon.mdx:22]: Good catch, @tw15egan !! Thank you. I'll fix now.

**Pull request merged**